### ISSUE TITLE
tr HTML tag is disapear on specefic condition

### ISSUE THREAD

**Issue by c0ncentus** [inikulin/parse5#345]
hi,

# Assert
```Javascript
const html = `<li id="Lala">qefe<p><o></o></p><ff><tr><i></i></tr></ff></li>`
console.log(serialize(parse(html)));
```

# Output 
```HTML
<html><head></head><body><li id="Lala">qefe<p><o></o></p><ff><i></i></ff></li></body></html>
```

What I'm expected from parser5

```HTML
<html><head></head><body><li id="Lala">qefe<p><o></o></p><ff><tr><i></i></tr></ff></li></body></html>
```

# Assert on global case :
```HTML
<table>
    <caption>Alien football stars</caption>
    <tr>
        <th scope="col">Player</th>
        <th scope="col">Gloobles</th>
        <th scope="col">Za'taak</th>
    </tr>
    <tr>
        <th scope="row">TR-7</th>
        <td>7</td>
        <td>4,569</td>
    </tr>
    <tr>
        <th scope="row">Khiresh Odo</th>
        <td>7</td>
        <td>7,223</td>
    </tr>
    <tr>
        <th scope="row">Mia Oolong</th>
        <td>9</td>
        <td>6,219</td>
    </tr>
</table>
```
and it work just as expected.

# Conclusion

> The HTML <tr> element defines a row of cells in a table. The row's cells can then be established using a mix of <td> (data cell) and <th> (header cell) elements.

if there are no throwing error of parser5 then it should admit we can do that.

So either throwing error for the html format or either make an update.

**Comment by wooorm** [comment]
That’s how browsers work, try this in a browser console:

```js
document.body.innerHTML = `<li id="Lala">qefe<p><o></o></p><ff><tr><i></i></tr></ff></li>`
document.body.innerHTML
"<li id=\"Lala\">qefe<p><o></o></p><ff><i></i></ff></li>"
```

**Comment by c0ncentus** [comment]
> ```js
> document.body.innerHTML
> "<li id=\"Lala\">qefe<p><o></o></p><ff><i></i></ff></li>"
> ```
# Mozilla Firefox ~v87
![image](https://user-images.githubusercontent.com/25374826/118268687-007e5f80-b4be-11eb-806c-ebbc8fd877bf.png)

It seems of what you say don't work on Mozilla Firefox, are you sure ?

**Comment by c0ncentus** [comment]
Ok I see 
![image](https://user-images.githubusercontent.com/25374826/118269424-032d8480-b4bf-11eb-8b81-de4e7d1fb216.png)

Yes but is not for what I'm expect ... I will wait for https://github.com/inikulin/parse5/issues/180  then ...

**Comment by frank-dspeed** [comment]
i think this can be closed? i also see no relation to #180

**Comment by fb55** [comment]
Thanks @wooorm for triaging!
